docs/narr/introduction.rst

```diff
@@ -139,103 +169,93 @@ decorators to localize the configuration.  For example:
+plain old ``print()`` calls, which will display to your console.
+
+Pyramid has a debug toolbar that allows you to see information about how your
+application is working right in your browser. See configuration, installed
+packages, SQL queries, logging statements and more.
 
-When developing a Pyramid application, several interactive features are
-available. Pyramid can automatically utilize changed templates when rendering
-pages and automatically restart the application to incorporate changed Python
-code. Plain old ``print()`` calls used for debugging can display to a console.
+When your application has an error, an interactive debugger allows you to poke
+around from your browser to find out what happened.
 
-Pyramid's debug toolbar comes activated when you use a Pyramid :term:`cookiecutter` to
-render a project.  This toolbar overlays your application in the browser, and
-allows you access to framework data, such as the routes configured, the last
-renderings performed, the current set of packages installed, SQLAlchemy queries
-run, logging data, and various other facts.  When an exception occurs, you can
-use its interactive debugger to poke around right in your browser to try to
-determine the cause of the exception.  It's handy.
+To use the Pyramid debug toolbar, build your project with a Pyramid
+:term:`cookiecutter`.
 
 Example: :ref:`debug_toolbar`.
 
-Debugging settings
-~~~~~~~~~~~~~~~~~~
+Debug with power
+~~~~~~~~~~~~~~~~
+
+When things go wrong, Pyramid gives you powerful ways to fix the problem.
 
-Pyramid has debugging settings that allow you to print Pyramid runtime
-information to the console when things aren't behaving as you're expecting. For
-example, you can turn on ``debug_notfound``, which prints an informative
-message to the console every time a URL does not match any view.  You can turn
-on ``debug_authorization``, which lets you know why a view execution was
-allowed or denied by printing a message to the console.  These features are
-useful for those WTF moments.
+You can configure Pyramid to print helpful information to the console. The
+``debug_notfound`` setting shows information about URLs that aren't matched.
+The ``debug_authorization`` setting provides helpful messages about why you
+aren't allowed to do what you just tried.
 
-There are also a number of commands that you can invoke within a Pyramid
-environment that allow you to introspect the configuration of your system.
-``proutes`` shows all configured routes for an application in the order they'll
-be evaluated for matching. ``pviews`` shows all configured views for any given
-URL.  These are also WTF-crushers in some circumstances.
+Pyramid also has command line tools to help you verify your configuration. You
+can use ``proutes`` and ``pviews`` to inspect how URLs are connected to your
+application code.
 
 Examples: :ref:`debug_authorization_section` and :ref:`command_line_chapter`.
```

Comment:
Add :doc:`pscripts/index` (I think that will work). We did not have these auto-documented when this was first written.